lib/datastore/entity.js

```diff
@@ -378,63 +396,79 @@ module.exports.propertyToValue = propertyToValue;
  * //   stringValue: 'Hi'
  * // }
  */
-function valueToProperty(v) {
-  var p = {};
-  if (v instanceof Boolean || typeof v === 'boolean') {
-    p.boolean_value = v;
-    return p;
-  }
-  if (v instanceof Int) {
-    p.integer_value = v.get();
-    return p;
-  }
-  if (v instanceof Double) {
-    p.double_value = v.get();
-    return p;
+function valueToProperty(value) {
```

Comment:
Renamed to `encodeValue`: https://github.com/GoogleCloudPlatform/gcloud-node/pull/897/files#diff-45459618925a219e865516ffadd74c28R413